app/.postgraphilerc.js

```diff
@@ -11,12 +11,6 @@ module.exports = {
       pgArchivedColumnName: 'archived_at',
       pgArchivedColumnImpliesVisible: false,
       pgArchivedRelations: false,
-      uploadFieldDefinitions: [
-        {
-          match: ({ schema, table, column, tags }) =>
-            table === 'attachment' && column === 'file',
-        },
-      ],
```

Comment:
It is unclear to me how this change is related to the bug. Removing this also alters our generated graphql schema, which I don't think is desirable.

Reply:
@matthieu-foucault  I had to remove  uploadFieldDefinitions here because it is also defined in app/backend/lib/graphql/index.ts, which references `app/.postgraphilerc.js`. It looks like uploadFieldDefinitions plugin was firing twice.
I tried to remove same block in app/backend/lib/graphql/index.ts but it resulted in runtime errors when file was uploaded.

Reply:
Although there is a bit of code that is duplicated, I don't think that the plugin is added twice, as the `uploadFieldDefinitions` value is being overridden here: https://github.com/bcgov/CONN-CCBC-portal/blob/c260600bb22a1804c769a5ad645fec8dc3ab238b/app/backend/lib/graphql/index.ts#L42-L51

As for writing tests, it is part of our team's definition of done, so I don't think it should be a separate ticket.

Reply:
It seems it was not overriden but added second time and that's what was causing duplication. I have no other explanation why issue was happening and disappeared after I removed duplicate in  app/.postgraphilerc.js
I totally agree with writing tests as part of our team's definition of done, but unfortunately, file upload feature was implemented without any tests.. and e2e Cypress tests have comments like `  // // Todo: file uploads` etc.